slices/openssl.yaml

```
@@ -23,6 +23,14 @@ slices:
       /usr/lib/ssl/openssl.cnf:
       /usr/lib/ssl/private:
 
+  # Include this slice and `ca-certificates_data` together to use the
+  # certificate bundle from `ca-certificates`.
+  # Missing this slice will cause applications using default certificates
+  # fail to verify local SSL certificates.
```

How about the following? (The link might trigger some linting warnings about line length)

```suggestion
  # Use this slice with "ca-certificates_data" as OpenSSL looks at
  # the following path for the default CA certificates file.
  # Ref: https://www.openssl.org/docs/manmaster/man3/SSL_CTX_load_verify_locations.html
```